logging/logging_test.go

```diff
@@ -717,6 +717,47 @@ func TestStandardLogger(t *testing.T) {
 	}
 }
 
+func TestStandardLoggerFromEntry(t *testing.T) {
+	initLogs() // Generate new testLogID
+	ctx := context.Background()
+	lg := client.Logger(testLogID)
+
+	slg := lg.StandardLoggerFromEntry(&logging.Entry{
+		Severity: logging.Info,
+		Trace:    "projects/P/traces/105445aa7843bc8bf206b120001000",
```

Comment:
More tests with different metadata filled out would be useful. Especially with payloads in the prototype entry, we want to make sure that's well covered by tests

Reply:
Done